docker-compose.yml

```diff
@@ -13,6 +13,8 @@ services:
   db:
     image: "mysql:5"
     environment:
+      XWIKI_VERSION: 7.4.3
```

Comment:
why is this under the "db" service? It shoud be under the "web" service IMO. WDYT?